### ISSUE TITLE
Fails to build on Fedora Rawhide

### ISSUE THREAD

**Issue by tim77** [WayfireWM/wayfire#1153]
**Describe the bug**
Seems like similar to https://github.com/WayfireWM/wf-config/issues/42.

```cpp
FAILED: subprojects/wf-touch/test/basic_test.p/basic_test.cpp.o 
g++ -Isubprojects/wf-touch/test/basic_test.p -Isubprojects/wf-touch/test -I../subprojects/wf-touch/test -Isubprojects/wf-touch -I../subprojects/wf-touch -I/usr/include -fdiagnostics-color=always -pipe -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wnon-virtual-dtor -Wextra -std=c++17 -O2 -flto=auto -ffat-lto-objects -fexceptions -g -grecord-gcc-switches -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -fstack-protector-strong -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -MD -MQ subprojects/wf-touch/test/basic_test.p/basic_test.cpp.o -MF subprojects/wf-touch/test/basic_test.p/basic_test.cpp.o.d -o subprojects/wf-touch/test/basic_test.p/basic_test.cpp.o -c ../subprojects/wf-touch/test/basic_test.cpp
In file included from /usr/include/signal.h:315,
                 from /usr/include/c++/11/csignal:42,
                 from ../subprojects/wf-touch/test/doctest.h:3241,
                 from ../subprojects/wf-touch/test/basic_test.cpp:2:
../subprojects/wf-touch/test/doctest.h:4403:45: error: size of array ‘altStackMem’ is not an integral constant-expression
 4403 |         static char             altStackMem[SIGSTKSZ];
      |                                             ^~~~~~~~
../subprojects/wf-touch/test/doctest.h:4453:48: error: size of array ‘altStackMem’ is not an integral constant-expression
 4453 |     char    FatalConditionHandler::altStackMem[SIGSTKSZ] = {};
      |                                                ^~~~~~~~
```

**To Reproduce**
Full build log and SRPM https://koji.fedoraproject.org/koji/taskinfo?taskID=66426941

**Expected behavior**
Should compile fine with latest software stacks.

**Wayfire version**
0.7.1

**Comment by ammen99** [comment]
Ah yes wf-touch also needs this. I'll fix it.

**Comment by ammen99** [comment]
@tim77 I just re-published the 0.7.1 version, it should contain the fix, can you verify?

**Comment by tim77** [comment]
@ammen99 built [successfully](https://koji.fedoraproject.org/koji/taskinfo?taskID=67209832). Thanks!
